File: server/src/main/java/com/e/shortform/domain/comment/res/CommentReply.java
package com.e.shortform.domain.comment.res;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class CommentReply {

    private Long id;
    private String commentReplyText;
    private Long commentReplyUserId;
    private Long commentReplyId;
    private String createAt;
    private String username;
    private String mention;
    private String profileImgSrc;
    private boolean social;
    private String provider;
    private Long likeCount;

}

File: client/src/entities/comment/ui/CommentReply.ts
export interface CommentReply {
    id: number;
    commentReplyText: string;
    commentReplyUserId: number;
    commentReplyId: number;
    createAt: string;
    username: string;
    mention: string;
    profileImgSrc: string;
    social: boolean;
    provider: string;
    likeCount: number;
}
